lib/Doctrine/ORM/Tools/Export/Driver/YamlExporter.php

```diff
@@ -244,8 +244,8 @@ protected function yamlDump($array, $inline = 2)
     }
 
     /**
-     * @psalm-param array{entityListeners: array<class-string, array<string, array{string}>>} $array
-     * @psalm-return array{entityListeners: array<class-string, array<string, array{string}>>}
+     * @psalm-param array<string, mixed> $array
+     * @psalm-return array<string, mixed>&array{entityListeners: array<class-string, array<string, array{string}>>}
```

Comment:
This is the best way I have found to say that this method doesn't super care about the shape of the array except that it must have an `entityListeners` key with that form.

Reply:
That's a perfectly valid syntax :)